client/landing/subscriptions/components/site-subscriptions-manager/site-row.tsx

```diff
@@ -137,7 +140,11 @@ const SiteRow = ( {
 					duration: 5000,
 					button: translate( 'Resubscribe' ),
 					onClick: () => {
-						resubscribe( { blog_id, url, doNotInvalidateSiteSubscriptions: true } );
+						if ( unsubscribeInProgress.current ) {
+							resubscribePending.current = true;
+						} else {
+							resubscribe( { blog_id, url, doNotInvalidateSiteSubscriptions: true } );
+						}
 						recordSiteResubscribed( {
```

Comment:
I noticed the resubscribe event is being tracked for every click, is it expected?
<img width="415" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3113712/366c013f-adb8-4775-9d2e-5e9c161aea30">